cypress/integration/10-database.js

```diff
@@ -62,6 +62,66 @@ describe('Database', () => {
       cy.get('#table-header > :nth-child(9)').contains('SEED BANKS');
     });
 
+    it('should select ALL the columns', () => {
+      cy.visit('/accessions');
+      cy.intercept('POST', '/api/v1/search').as('search');
+      cy.intercept('POST', '/api/v1/seedbank/values').as('values');
+
+      cy.get('#more-options').click();
```

Comment:
@constanzauanini can we do this programmatically by search for elements of a certain type? this will be cleaner and won't need any updates.

Reply:
Sounds good!